providers/config_default.yaml

```diff
@@ -155,6 +155,18 @@ VSPHERE_AZ_0:
 VSPHERE_AZ_1:
 VSPHERE_AZ_2:
 
+#! Settings to provide netpermissions for vpshere-csi file volumes
+#! example:
+#! NETPERMISSIONS:
+#!   A:
+#!     ips: "*"
+#!     permissions: READ_WRITE
+#!     rootsquash: false
+NETPERMISSIONS:
+
+#! The namespace csi components are to be deployed in
+CSI_NAMESPACE: ""
```

Comment:
CSI namespace is controlled by addon manager and should not expose to users

Reply:
will remove